stdlib/testing/testdata/aggregate_empty_window.flux

```diff
@@ -55,4 +55,3 @@ test aggregate_window_empty = () => ({
             |> aggregateWindow(every: 30s, fn: sum),
 })
 
-testing.run(case: aggregate_window_empty)
```

Comment:
My understanding is that now the `test` keyword is an implicit call to testing.run. Where is that happening? I didn't see it in this code.

Reply:
The call to `testing.run` is added to the end of the ast right here https://github.com/influxdata/flux/blob/017b3bd84fb1bdac2e3e8ae836fd1f1f4900bda7/internal/cmd/builtin/cmd/generate.go#L257 .